Content.Client/Changelog/ChangelogManager.cs

```diff
@@ -195,6 +195,9 @@ public sealed partial class ChangelogEntry
 
             [DataField("changes")]
             public List<ChangelogChange> Changes { get; private set; } = default!;
+
+            [DataField("labels")]
+            public List<string> Labels { get; private set; } = default!;
```

Comment:
```suggestion
            [DataField]
            public List<string> Labels { get; private set; } = [];
```
Also should probably have docs.